Description: No longer call rmic to build with OpenJDK 15 and later
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -211,11 +211,6 @@
       </src>
     </javac>
 
-    <rmic
-      base="build/classes" includes="freemarker/debug/impl/Rmi*Impl.class"
-      classpathref="ivy.dep"
-      verify="yes" stubversion="1.2"
-    />
 
     <ivy:cachepath conf="build.j2se1.5" pathid="ivy.dep.j2se1.5" />
     <javac srcdir="src/main/java" destdir="build/classes" deprecation="off" 
@@ -277,11 +272,6 @@
         freemarker/ext/jython/_Jython22VersionAdapter.java"
     />
     
-    <rmic base="build/classes" classpathref="ivy.dep"
-      includes="src/main/java/freemarker/debug/impl/Rmi*Impl.class"
-      verify="yes" stubversion="1.2"
-    />
-    
     <!-- We don't have this file in 2.4.X... yet?
     <copy
       file="build/classes/freemarker/core/SecureRendererImpl.class"
